packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1082,6 +1082,40 @@ export class Vector3 {
         return result.copyFromFloats(this._x * scale, this._y * scale, this._z * scale);
     }
 
+    /**
+     * Rotates the vector using the given unit quaternion and stores the new vector in result
+     * @param q the unit quaternion representing the rotation
+     * @param result the output vector
+     */
+    public applyRotationQuaternionToRef(q: Quaternion, result: Vector3): Vector3 {

```

Comment:
Can you add @returns annotation in the three functions ?

Reply:
Will do ! Don't know why it wasn't auto generated

Reply:
All done!